src/commonTest/kotlin/exchange.dydx.abacus/tests/mock/processor/wallet/account/FillProcessorMock.kt

```diff
@@ -0,0 +1,18 @@
+package exchange.dydx.abacus.tests.mock.processor.wallet.account
+
+import exchange.dydx.abacus.output.SubaccountFill
+import exchange.dydx.abacus.processor.wallet.account.FillProcessorProtocol
+import indexer.codegen.IndexerFillResponseObject
+
+class FillProcessorMock : FillProcessorProtocol {
```

Comment:
super-nit: Personally I prefer the term `Fake` for implementations written by us. Colloquially, I understand `Mock` to refer to implementations generated by a mocking framework.